### PR TITLE
Happy thoughts - Joel

### PR DESCRIPTION
**Pull request by JHarde**
Link to my deployed project:

My reflections on how this week's project turned out:

Things I'd like to have clarified or explained in more detail:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,11 @@
...
 
 ## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+My biggest problem was making the like button work correctly. Initially it posted the information but didn't fetch it back until I refreshed the page. After a while I manage to see the updated change, however I got a bad request 400 and it liked all the posts. After inspecting the code for a while I found the problem and was able to solve it.
 
 ## View it live
 
-Every project should be deployed somewhere. Be sure to include the link to the deployed project so that the viewer can click around and see what it's all about.
+https://pensive-ritchie-543d02.netlify.app/
```
Comment (MayaBlixt) [inline README.md:11]: Maybe add a section about ## Limitations where you can add more information if the website is responsive or not.

**Review by MayaBlixt — APPROVED** [review]
Nice job Joel 👍 😄  Can tell that you've taken you're time to work through the project and really understand what you're doing. I like how you've structured your components, simple and clear to follow.

It it's up to you if you want to implement any of the changes that I've suggested 😊

File: code/src/App.js [hunk 2]
```diff
@@ -1,9 +1,53 @@
...
+  };
+
+  const reachMessageInput = (newMessage) => {
+    fetch(MESSAGES_URL, {
+      method: "POST",
+      headers: { "Content-type": "application/json" },
+      body: JSON.stringify({ message: newMessage }),
+    })
+      .then(() => fetchMessages())
+      .catch((err) => console.log("error:", err));
```
Comment (EmOlsz) [inline code/src/App.js:28]: `console.error` would be more appropriate here

File: code/src/App.js [hunk 3]
```diff
@@ -1,9 +1,53 @@
...
+  };
+
   return (
-    <div>
-      Find me in src/app.js!
+    <div className="main-container">
+      <MessageInput onMessageChange={reachMessageInput} />
+
+      <MessageList
+        key={messages._id}
```
Comment (EmOlsz) [inline code/src/App.js:47]: You don't need `key` attribute here, we are not performing any loop

**Comment by JHarde** [comment]
Have tried to implement some of the changes now, let me know if more is needed! 👍

**Comment by EmOlsz** [comment]
It's ok now, thanks Joel, approved!

**Pull request closed without merging**